### PR TITLE
sendJiraRequest: Log error content

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
Instead of
```
400 (Bad Request): 
```
report
```
400 (Bad Request): 
{
  "errorMessages": [],
  "errors": {
    "project": "Specify a valid project ID or key"
  }
}
```

### CODE REVIEW

File: lib/Constants.ts [hunk 1]
```diff
@@ -1,2 +1,3 @@
 export const JIRA_ISSUE_PATTERN: RegExp = /[A-Z][A-Z0-9]*-\d+/g;
+// Use https://sonarsource-sandbox-608.atlassian.net for testing
```
Comment (pavel-mikula-sonarsource) [inline lib/Constants.ts:2]: Note for future generations an their dev branches

Comment (sebastien-marichal) [reply lib/Constants.ts:2]: Can you make this info public?

**Review by sebastien-marichal — APPROVED** [review]
LGTM!

File: lib/Constants.ts (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply lib/Constants.ts:2]: What do you mean?

Comment (sebastien-marichal) [reply lib/Constants.ts:2]: I was wondering if it is ok to have our sandbox url in a public repository.

Comment (pavel-mikula-sonarsource) [reply lib/Constants.ts:2]: Ok, I can remove it. The instance itself is publicly accessible, with all its data. So there should not be an issue

**Pull request merged**